docs/variants/talos_2/hardware-matrix.md

```diff
@@ -5,23 +5,20 @@
 This document describes the hardware configuration used for validation of the
 coreboot port on the Talos II platform.
 
-## Board No. 1
+## Talos II
 
 | Component                      | Description                                                   |
 |--------------------------------|---------------------------------------------------------------|
-| **CPU**                        | TBD                                                           |
-|                                | TBD                                                           |
+| **CPU**                        | IBM POWER9 “Sforza”                                           |
 |                                | CPU Cooler                                                    |
-| **RAM**                        | TBD                                                           |
-| **Flash memory**               | TBD                                                           |
+| **RAM**                        | Crucial CT8G4RF88266                                          |
```

Comment:
Is this the only one RAM module mounted in the platform?

Reply:
Currently yes